src/containers/WorkspaceRecordingView.tsx

```diff
@@ -0,0 +1,67 @@
+import { Grid } from '@material-ui/core';
+import React, { FunctionComponent } from 'react';
+import { WorkspaceInfo } from '../AppContainer';
+import RecordingInfoView from '../components/RecordingInfoView';
+import SortingsView from '../components/SortingsView';
+import { useRecordingInfo } from '../extensions/common/getRecordingInfo';
+import { createCalculationPool } from '../extensions/common/hither';
+import { Plugins, RecordingSelectionAction } from '../extensions/extensionInterface';
+import sortByPriority from '../extensions/sortByPriority';
+import { Recording } from '../reducers/recordings';
+import { Sorting } from '../reducers/sortings';
+import { Expandable } from './SortingView';
+import { WorkspaceRouteDispatch } from './WorkspaceView';
+
+interface Props {
+  recording: Recording
+  sortings: Sorting[]
+  workspaceInfo: WorkspaceInfo
+  plugins: Plugins
+  workspaceRouteDispatch: WorkspaceRouteDispatch
+}
+
+const calculationPool = createCalculationPool({maxSimultaneous: 6})
+
+const WorkspaceRecordingView: FunctionComponent<Props> = ({ recording, sortings, workspaceInfo, plugins, workspaceRouteDispatch }) => {
+  const { readOnly } = workspaceInfo;
+  const recordingInfo = useRecordingInfo(recording.recordingObject)
+  if (!recordingInfo) return <div>Loading recording info</div>
+
+  const handleImportSortings = () => {
+    console.info('not yet implemented')
```

Comment:
This is eventually planned to drive a python script that imports the sortings?

Reply:
it will display python snippets for showing how to import sortings.